src/storage/types.ts

```diff
@@ -6,6 +6,8 @@ export interface TX {
   address: string;
   inputs: Input[];
   outputs: Output[];
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  fees?: any;
```

Comment:
why fee ?
its already deductible from inputs and outputs

Reply:
Yes but it's actually provided by the explorer, so for sync it's better to just get the value.